code/modules/mob/living/carbon/human/species.dm

```diff
@@ -710,6 +714,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		if(!source.dna.features["spines"] || source.dna.features["spines"] == "None" || source.wear_suit && (source.wear_suit.flags_inv & HIDEJUMPSUIT))
 			bodyparts_to_add -= "spines"
 
+	if(mutant_bodyparts["spines_vox"])
```

Comment:
If these are an organ they do not need to be here, bodypart code draws them.

Reply:
but they're not

Reply:
didnt know if they were or not